rusk-abi/tests/lib.rs

```diff
@@ -198,6 +199,61 @@ mod module_tests {
         );
     }
 
+    #[test]
+    fn bls_signature() {
+        let host = HostFnTest::new();
+
+        let code = include_bytes!(
+            "../../target/wasm32-unknown-unknown/release/host_fn.wasm"
+        );
+
+        let contract = Contract::new(host, code.to_vec());
+
+        let rusk_mod = RuskModule::new(&PUB_PARAMS);
+        let mut network = NetworkState::default();
+        network.register_host_module(rusk_mod);
+
+        let contract_id = network.deploy(contract).unwrap();
+
+        let mut gas = GasMeter::with_limit(1_000_000_000);
+
+        let message = b"some-message".to_vec();
+
+        let sk = BlsSecretKey::random(&mut rand_core::OsRng);
+        let pk = (&sk).into();
+        let apk = BlsPublicKey::from(&sk);
```

Comment:
Any particular reason in create the APK from SecretKey instead of from PublicKey?

Reply:
Ok, I didn't notice this #52
My fault, I flag this comment as resolved